RevenueCatUI/UIKit/PaywallFooterViewController.swift

```diff
@@ -51,18 +53,21 @@ public final class PaywallFooterViewController: PaywallViewController {
     public init(offeringIdentifier: String, fontName: String) {
         super.init(content: .offeringIdentifier(offeringIdentifier),
                    fonts: CustomPaywallFontProvider(fontName: fontName),
-                   displayCloseButton: false)
+                   displayCloseButton: false,
+                   dismissalRequest: nil)
     }
 
     @available(*, unavailable)
     override init(
         content: PaywallViewConfiguration.Content,
         fonts: PaywallFontProvider,
-        displayCloseButton: Bool = false
+        displayCloseButton: Bool = false,
+        dismissalRequest: ((_ controller: PaywallViewController) -> Void)? = nil
```

Comment:
Hmm while I think we need to add it to the other constructors here, why do we add it to the unavailable constructor?

Reply:
oh actually, this one is overriden from `PaywallViewController`, so it's required :/